create.ps1

```diff
@@ -1,110 +1,50 @@
-#
-# Version: 1.1.0
-########################################
-# Initialize default values
-$config = $configuration | ConvertFrom-Json
-$p = $person | ConvertFrom-Json
-$success = $false
-$auditLogs = [System.Collections.Generic.List[PSCustomObject]]::new()
-
-#region support functions
-function Get-FullName {
-    Param (
-        [object]$person
-    )
-
-    if ([string]::IsNullOrEmpty($p.Name.Nickname)) { $calcFirstName = $p.Name.GivenName } else { $calcFirstName = $p.Name.Nickname }
+# PowerShell V2
+#################################################
 
-    $calcFullName = $calcFirstName + ' '
+# Enable TLS1.2
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor [System.Net.SecurityProtocolType]::Tls12
 
-    switch ($person.Name.Convention) {
-        'B' { 
-            if (-not [string]::IsNullOrEmpty($person.Name.familyNamePrefix)) {
-                $calcFullName = $calcFullName + $person.Name.familyNamePrefix + ' '
-            }
-            $calcFullName = $calcFullName + $person.Name.FamilyName
-            break 
-        }
-        'P' { 
-            if (-not [string]::IsNullOrEmpty($person.Name.familyNamePartnerPrefix)) {
-                $calcFullName = $calcFullName + $person.Name.familyNamePartnerPrefix + ' '
-            }
-            $calcFullName = $calcFullName + $person.Name.FamilyNamePartner
-            break 
-        }
-        'BP' { 
-            if (-not [string]::IsNullOrEmpty($person.Name.familyNamePrefix)) {
-                $calcFullName = $calcFullName + $person.Name.familyNamePrefix + ' '
-            }
-            $calcFullName = $calcFullName + $person.Name.FamilyName + ' - '
-            if (-not [string]::IsNullOrEmpty($person.Name.familyNamePartnerPrefix)) {
-                $calcFullName = $calcFullName + $person.Name.familyNamePartnerPrefix + ' '
-            }
-            $calcFullName = $calcFullName + $person.Name.FamilyNamePartner
-            break 
-        }
-        'PB' { 
-            if (-not [string]::IsNullOrEmpty($person.Name.familyNamePartnerPrefix)) {
-                $calcFullName = $calcFullName + $person.Name.familyNamePartnerPrefix + ' '
-            }
-            $calcFullName = $calcFullName + $person.Name.FamilyNamePartner + ' - '
-            if (-not [string]::IsNullOrEmpty($person.Name.familyNamePrefix)) {
-                $calcFullName = $calcFullName + $person.Name.familyNamePrefix + ' '
-            }
-            $calcFullName = $calcFullName + $person.Name.FamilyName
-            break 
-        }
-        Default {
-            if (-not [string]::IsNullOrEmpty($person.Name.familyNamePrefix)) {
-                $calcFullName = $calcFullName + $person.Name.familyNamePrefix + ' '
-            }
-            $calcFullName = $calcFullName + $person.Name.FamilyName
-            break 
-        }
-    } 
-    return $calcFullName
+# Set debug logging
+switch ($actionContext.Configuration.isDebug) {
```

Comment:
In principe, als we gebruik maken van `Write-Information` / `Write-Warning` bepaalt HelloID zelf wanneer een message zichtbaar is. (Alleen tijdens preview / retry). Zou je eventueel nog kunnen ombouwen. Dan zou ook de toggle uit de configuration kunnen.

Reply:
De reden waarom ik het graag wil behouden (andere ook):
Write-warning = wordt altijd gelogd
Write-information = wordt altijd gelogd bij manual / retry (kan je klant dus extra informatie geven, maar niet informatie geven waar ze niks mee kunnen)
Write-verbose = Debug logging voor consultant / support. Dus de uitgebreide logging zoals de splat uitprinten